Sources/JavaScriptKit/JSValueConvertible.swift

```diff
@@ -57,69 +69,63 @@ extension JSObjectRef: JSValueConvertible {
     // from `JSFunctionRef`
 }
 
-private let Object = JSObjectRef.global.Object.function!
-
-extension Dictionary where Value: JSValueConvertible, Key == String {
+extension Optional: JSValueConvertible where Wrapped: JSValueConvertible {
+    
     public func jsValue() -> JSValue {
-        Swift.Dictionary<Key, JSValueConvertible>.jsValue(self)()
+        switch self {
+        case .none:
+            return .null
+        case let .some(value):
+            return value.jsValue()
+        }
     }
 }
 
-extension Dictionary: JSValueConvertible where Value == JSValueConvertible, Key == String {
+extension Dictionary where Value: JSValueConvertible, Key == String {
     public func jsValue() -> JSValue {
-        let object = Object.new()
-        for (key, value) in self {
-            object.set(key, value.jsValue())
-        }
-        return .object(object)
+        Swift.Dictionary<Key, JSValueConvertible>.jsValue(self)()
     }
 }
 
-private let Array = JSObjectRef.global.Array.function!
-
-extension Array where Element: JSValueConvertible {
+extension Dictionary: JSValueConvertible where Value == JSValueConvertible, Key == String {
     public func jsValue() -> JSValue {
-        Swift.Array<JSValueConvertible>.jsValue(self)()
+        let object = JSObject()
+        self.forEach { object[$0.key] = $0.value.jsValue() }
+        return .object(object.jsObject)
     }
 }
 
-extension Array: JSValueConvertible where Element == JSValueConvertible {
```

Comment:
Please keep this extension for Element == JSValueConvertible to be available for existential